indico/web/client/styles/modules/_papers.scss

```diff
@@ -46,6 +46,10 @@
     label {
         font-weight: normal;
     }
+
+    .text-not-important {
```

Comment:
Wouldn't it be cleaner to define a new class? (inside `#assign-role-table` ofc)
```
    .affiliation {
        font-size: 0.8em;
        color: $light-black;
    }
```
@ThiefMaster

Reply:
yes, agreed!